Sources/Sentry/Public/SentrySpanProtocol.h

```diff
@@ -87,6 +87,28 @@ NS_SWIFT_NAME(Span)
  */
 - (void)removeTagForKey:(NSString *)key NS_SWIFT_NAME(removeTag(key:));
 
+- (void)setMeasurement:(NSString *)name
```

Comment:
Should we add some comments briefly explaining which units can be used with which overload or do we expect people to simply drill down on the enum they can pass?

Reply:
I will add comments for sure. Thanks for pointing that out.